### PR TITLE
Add support for migrating to remote db

### PR DESCRIPTION
**Pull request by h4xr**

### CODE REVIEW

File: playbooks/experiments/roles/pg_remote_db_populate/tasks/main.yaml [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+    become: True
+    become_user: postgres
+    delegate_to: "{{ pgsql_host }}"
+  - name: "Create database for candlepin"
+    shell:
+            psql -U postgres -c "CREATE DATABASE candlepin OWNER candlepin"
+    become: True
+    become_user: postgres
+    delegate_to: "{{ pgsql_host }}"
+  - name: "Get the foreman dump from satellite"
```
Comment (jhutar) [inline playbooks/experiments/roles/pg_remote_db_populate/tasks/main.yaml]: Could we copy directly to /tmp/?

File: playbooks/experiments/roles/pg_remote_db_populate/tasks/main.yaml [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+         - foreman
+         - candlepin
+    delegate_to: "{{ pgsql_host }}"
+  - name: "Enable pg_stat_statements"
+    shell:
+            psql -U postgres -c "CREATE EXTENSION pg_stat_statements"
+    become: True
+    become_user: postgres
+    delegate_to: "{{ pgsql_host }}"
+...
```
Comment (jhutar) [inline playbooks/experiments/roles/pg_remote_db_populate/tasks/main.yaml]: IMO we should delete (cleanup) these files after import

File: playbooks/experiments/README.md [hunk 3]
```diff
@@ -0,0 +1,7 @@
+# Experiments
```
Comment (jhutar) [inline playbooks/experiments/README.md]: Good idea!

File: playbooks/experiments/roles/pg_remote_db_setup/tasks/main.yaml [hunk 4]
```diff
@@ -0,0 +1,104 @@
...
+            service postgresql initdb
+    delegate_to: "{{ pgsql_host }}"
+  - name: "Configure pgsql"
+    lineinfile:
+          dest: /var/lib/pgsql/data/postgresql.conf
+          regexp: "^#listen_address"
+          line: "listen_address: '0.0.0.0'"
+          state: present
+    delegate_to: "{{ pgsql_host }}"
+  - lineinfile:
```
Comment (jhutar) [inline playbooks/experiments/roles/pg_remote_db_setup/tasks/main.yaml]: Could we have "name" for at least some of these lineinfile if they are your customizations? If this is only something which mirrors Sat6 defaultsetup, then OK

File: playbooks/experiments/roles/pg_remote_db_setup/tasks/main.yaml [hunk 5]
```diff
@@ -0,0 +1,104 @@
...
+          line: "stats_temp_directory = pg_stat_temp"
+          state: present
+    delegate_to: "{{ pgsql_host }}"
+  - name: "Assure that postgresql restarts correctly"
+    service:
+       name: postgresql
+       state: restarted
+    delegate_to: "{{ pgsql_host }}"
+  - set_fact:
+          satellite_ip: "{{ ansible_eth0.ipv4.address }}"
```
Comment (jhutar) [inline playbooks/experiments/roles/pg_remote_db_setup/tasks/main.yaml]: I do not understand why this should return satellite's IP? Isn't it going to return external PostgreSQL's server IP?

File: playbooks/experiments/roles/pg_remote_db_populate/tasks/main.yaml (hunk 1 above)
Comment (h4xr) [inline playbooks/experiments/roles/pg_remote_db_populate/tasks/main.yaml]: Initially was trying to copy directly to temp, but when tried to login to the server using ssh, the files didn't seem to be there

File: playbooks/experiments/roles/pg_remote_db_populate/tasks/main.yaml (hunk 2 above)
Comment (h4xr) [inline playbooks/experiments/roles/pg_remote_db_populate/tasks/main.yaml]: Yups, might have forgot. Will add :)

File: playbooks/experiments/roles/pg_remote_db_setup/tasks/main.yaml (hunk 5 above)
Comment (h4xr) [inline playbooks/experiments/roles/pg_remote_db_setup/tasks/main.yaml]: Actually, In Postgres we need to authorize the remote server(in our case, the satellite server) to connect to Postgres. This is being done by taking the up address of the satellite 6 and putting it inside pg_hba.conf so as to authorize the connection attempts made by satellite to our remote database.

**Comment by h4xr** [comment]
@jhutar I'm still trying to squash some bugs so, will be completing this probably by this week. 
Also, the ansible.cfg seems to be getting merged because of a previous commit I made

**Comment by jhutar** [comment]
Forgot about this

**Pull request merged**